lucene/backward-codecs/src/test/org/apache/lucene/backward_index/TestBackwardsCompatibility.java

```diff
@@ -1469,6 +1558,10 @@ private void addDoc(IndexWriter writer, int id) throws IOException {
     customType4.setStoreTermVectorOffsets(true);
     customType4.setIndexOptions(IndexOptions.DOCS_AND_FREQS);
     doc.add(new Field("content6", "here is more content with aaa aaa aaa", customType4));
+
+    float[] vector = {0.1f, 0.1f, 0.1f + id};
```

Comment:
nit: maybe vary the numbers a bit more, e.g.
```suggestion
    float[] vector = {0.2f, -0.1f, 0.1f + id};
```